mlrun/runtimes/sparkjob/abstract.py

```diff
@@ -696,7 +696,7 @@ def with_requests(self, mem=None, cpu=None, patch: bool = False):
 
     def gpus(self, gpus, gpu_type="nvidia.com/gpu"):
         raise NotImplementedError(
-            "In spark runtimes, please use with_driver_requests & with_executor_requests"
+            "In spark runtimes, please use with_driver_limits & with_executor_limits"
```

Comment:
```suggestion
            "In spark runtimes, use `with_driver_limits` & `with_executor_limits`"
```

No need for `please` here.